src/Sulu/Bundle/PreviewBundle/Resources/js/containers/Preview/Preview.js

```diff
@@ -235,6 +237,32 @@ class Preview extends React.Component<Props> {
         return this.iframeRef.contentDocument;
     };
 
+    getPreviewWindow= (): ?Window => {
+        if (this.previewWindow) {
+            return this.previewWindow;
+        }
+
+        if (!(this.iframeRef instanceof HTMLIFrameElement)) {
```

Comment:
is this working as expected mostly `instanceof` makes problem if there is a build. Is there no other way to check if its a window or an iframe, which is already used somewhere /cc @wachterjohannes

Reply:
@alexander-schranz we already used `instanceof` and i thought that there is no problem until now. what are you thinking about? (see https://github.com/sulu/sulu/blob/2.x/src/Sulu/Bundle/PreviewBundle/Resources/js/containers/Preview/Preview.js#L232)

Reply:
Sounds good then it works with native objects without problems. Transpiled classes make problems to be checkt with `instanceof`.